packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -39,17 +43,47 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   assert(tools, 'no tools');
   const terms = zcf.getTerms();
   assert('USDC' in terms.brands, 'no USDC brand');
-  assert('PoolShares' in terms.brands, 'no PoolShares brand');
+
+  const { makeRecorderKit } = prepareRecorderKitMakers(
+    zone.mapStore('vstorage'),
```

Comment:
please add a snapshot test of contract baggage to show where all this stuff is going

https://github.com/Agoric/agoric-sdk/blob/84d84f48dcf9c2c101ef7652d12939d5e43c22e7/packages/orchestration/test/examples/send-anywhere.test.ts#L230-L253